@here/harp-mapview/lib/Utils.ts

```diff
@@ -317,8 +319,9 @@ export namespace MapViewUtils {
         } else if (deltaTilt <= 0) {
             // Reducing the tilt isn't clamped (apart from above).
             return deltaTilt;
-        } else if (mapTargetWorld.equals(rotationTargetWorld)) {
-            // When the rotation target is the center, then we have a simple formula
+        } else if (mapTargetWorld.equals(rotationTargetWorld) || offsetY < 0) {
+            // When the rotation target is the center, or the offsetY is < 0, i.e. the angle at the
+            // `mapTargetWorld` is always bigger, then we have a simple formula
```

Comment:
>or the offsetY is < 0, i.e. the angle at the
// `mapTargetWorld` is always bigger, then we have a simple formula

That's not necessarily true for globe. Imaging you see the whole globe with tilt 0 at the target, and you pick a rotation center below the target near the horizon.

Reply:
In that case it has -ve tilt (relative to the rotation target)